### DISCUSSION TITLE
Proxy support for ipp requests

### DISCUSSION THREAD

**Discussion by amit-04** [OpenPrinting/libcups#18] [Q&A]
Hi
I was working on a project where I was using libcups to connect to infrastructure printer. I am wondering how to make IPP calls in proxy environment(with proxy server/username/password). I have checked the documentation and function but I could not find anything. Is there any example/code snippet or if somebody can let me know about it.

Thanks
Amit

**Comment by michaelrsweet** [comment]
Look at the "ippproxy.c" source in the ISTO PWG's [ippsample](https://github.com/istopwg/ippsample) project.
